Userland/Libraries/LibHTTP/HttpRequest.cpp

```diff
@@ -12,32 +12,37 @@
 
 namespace HTTP {
 
-String HttpRequest::method_name() const
+String method_name(HttpRequest::Method method)
```

Comment:
Can we call this method_to_string or something?

Reply:
That would make a lot of sense, yes.